<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Spell Checker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Video Spell Checker</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="subtitleLabel">
      <property name="styleSheet">
       <string notr="true">color: gray;</string>
      </property>
      <property name="text">
       <string>Проверка орфографии в видео (русский/английский)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="fileLayout">
      <item>
       <widget class="QLabel" name="fileLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Видео файл:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileInput">
        <property name="placeholderText">
         <string>Выберите видео файл...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Обзор...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="outputLayout">
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Папка вывода:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputDirInput">
        <property name="placeholderText">
         <string>Папка для результатов...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseOutputButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Обзор...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="timeLayout">
      <item>
       <widget class="QLabel" name="intervalLabel">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Интервал (сек):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="intervalInput">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="startLabel">
        <property name="text">
         <string>Начало [час:мин:сек]:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="startTimeInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::Section::SecondSection</enum>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="endLabel">
        <property name="text">
         <string>Конец [час:мин:сек]:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="endTimeInput">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
        </property>
        <property name="currentSection">
         <enum>QDateTimeEdit::Section::SecondSection</enum>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
        <property name="time">
         <time>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
         </time>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: palette(button);
    color: palette(button-text);
    font-size: 14px;
    font-weight: bold;
    border: 2px solid palette(mid);
    border-radius: 5px;
    padding: 8px;
}
QPushButton:hover {
    background-color: palette(light);
    border: 2px solid palette(highlight);
}
QPushButton:pressed {
    background-color: palette(mid);
}
QPushButton:disabled {
    background-color: palette(window);
    color: palette(mid);
    border: 2px solid palette(mid);
}</string>
      </property>
      <property name="text">
       <string>Начать проверку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="styleSheet">
       <string notr="true">QProgressBar {
    background-color: palette(base);
    color: palette(text);
    border: 1px solid palette(mid);
    border-radius: 5px;
    text-align: center;
    height: 25px;
}
QProgressBar::chunk {
    background-color: palette(highlight);
    border-radius: 4px;
}</string>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="format">
       <string>%p% - Обработано кадров</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="logLayout" stretch="0,1">
        <item>
         <widget class="QLabel" name="logLabel">
          <property name="text">
           <string>Лог обработки:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="logOutput">
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QTextEdit {
    border: 1px solid palette(mid);
    border-radius: 5px;
    padding: 10px;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="previewLayout">
        <item>
         <widget class="QLabel" name="previewLabel">
          <property name="text">
           <string>Текущий кадр:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="framePreview">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>480</width>
            <height>360</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
    background-color: palette(base);
    color: palette(text);
    border: 1px solid palette(mid);
    border-radius: 5px;
}</string>
          </property>
          <property name="text">
           <string>Ожидание обработки...</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpenDictionary"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpenDictionary">
   <property name="text">
    <string>Открыть пользовательский словарь...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>fileInput</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>intervalInput</tabstop>
  <tabstop>startTimeInput</tabstop>
  <tabstop>endTimeInput</tabstop>
  <tabstop>startButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
